pkg/client/client.go

```diff
@@ -173,7 +173,7 @@ func Initialize(
 	)
 
 	// Watch for new keeps creation.
-	ethereumChain.OnBondedECDSAKeepCreated(func(event *eth.BondedECDSAKeepCreatedEvent) {
+	_, err := ethereumChain.OnBondedECDSAKeepCreated(func(event *eth.BondedECDSAKeepCreatedEvent) {
```

Comment:
Can we deal with the error returned from `OnBondedECDSAKeepCreated` the same way as we did it in `keep-core`?  Explanation: https://github.com/keep-network/keep-core/pull/1996#issuecomment-679062032